### PR TITLE
Prepare release

### PR DESCRIPTION
**Pull request by jeetiss**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,9 +9,26 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
...
+### Changed
+
+- Migrate from Coffee to Javascript. See [#567][567].
+- Es modules instead namespaces. See [#569][569].
+
 ### Fixed
 
-- Spanish locale was updated.
+- Improve accessibility. Thanks to @erin-doyle. See [#594][594] and [#595][595].
+- Spanish locale was updated. Thanks to @HerrLindner. See [#550][550].
```
Comment (nd0ut) [inline CHANGELOG.md]:
```suggestion
* Spanish locale was updated. Thanks to @HerrLindner. See [#550][550].
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -9,9 +9,26 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
...
+- German locale was updated. Thanks to @thomasklemm. See [#580][580].
```
Comment (nd0ut) [inline CHANGELOG.md]:
```suggestion
* German locale was updated. Thanks to @thomasklemm. See [#580][580].
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -9,9 +9,31 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 [Unreleased]: https://github.com/uploadcare/uploadcare-widget/compare/v3.7.9...HEAD
 
+## [3.8.0]
+
+### Changed
+
+* Migrate from Coffee to Javascript. See [#567][567].
```
Comment (dayton1987) [inline CHANGELOG.md]: Migrated from CoffeScript to <native> JavaScript. See ...

File: CHANGELOG.md [hunk 4]
```diff
@@ -9,9 +9,31 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
...
+* Es modules instead of namespaces. See [#569][569].
```
Comment (dayton1987) [inline CHANGELOG.md]: Switched to ES Modules instead of namespaces...

File: CHANGELOG.md [hunk 5]
```diff
@@ -9,9 +9,31 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
...
+* Migrate templates to tagged HTML strings from EJS. See [#603][603].
```
Comment (dayton1987) [inline CHANGELOG.md]: Migrated ...

File: CHANGELOG.md [hunk 6]
```diff
@@ -9,9 +9,31 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
...
+
 ### Fixed
 
-- Spanish locale was updated.
+* Improve accessibility. Big thanks to @erin-doyle. See [#594][594], [#595][595] and [#611][611].
```
Comment (dayton1987) [inline CHANGELOG.md]: Improved widget accessibility...

File: CHANGELOG.md [hunk 7]
```diff
@@ -9,9 +9,31 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
...
+* Spanish locale was updated. Thanks to @HerrLindner. See [#550][550].
```
Comment (dayton1987) [inline CHANGELOG.md]: Updated Spanish locale

File: CHANGELOG.md [hunk 8]
```diff
@@ -9,9 +9,31 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
...
+* German locale was updated. Thanks to @thomasklemm. See [#580][580].
```
Comment (dayton1987) [inline CHANGELOG.md]: Updated German locale

File: CHANGELOG.md [hunk 9]
```diff
@@ -9,9 +9,31 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
...
+* Fix effects tab locales. See [#613][613].
```
Comment (dayton1987) [inline CHANGELOG.md]: Fixed

**Pull request merged**